tests/robot-tests/tests/admin/bau/create-amendment-check-permalink.robot

```diff
@@ -0,0 +1,198 @@
+    user clicks checkbox    England
+    user clicks element     id:locationFiltersForm-submit
+    user waits until h2 is visible  Choose time period
+    user checks previous table tool step contains  3   National     England
+
+Select Start date and End date
+    [Tags]  HappyPath
+    user selects from list by label  id:timePeriodForm-start   2020 Week 14
+    user selects from list by label  id:timePeriodForm-end     2020 Week 14
+    user clicks element     css:#timePeriodForm-submit
+    user waits until h2 is visible  Choose your filters
+    user waits until page contains element   id:filtersForm-indicators
+    user checks previous table tool step contains  4    Start date    2020 Week 14
+    user checks previous table tool step contains  4    End date      2020 Week 14
+
+Select Indicators
+    [Tags]  HappyPath
+    user clicks indicator checkbox   Number of open settings
+
+Select Filters
+    [Tags]   HappyPath
+    user opens details dropdown   Date
+    user clicks category checkbox   Date   30/03/2020
+    user checks category checkbox is checked   Date   30/03/2020
+
+Create table
+    [Tags]  HappyPath
+    user clicks element     css:#filtersForm-submit
+    user waits until results table appears     60
+
+Validate results table column headings
+    [Tags]  HappyPath
+    user checks table column heading contains  1   1   2020 Week 14
+
+Generate the peramlink
+  [Tags]  HappyPath
+  user clicks button  Generate permanent link
+  user waits until page contains element  xpath://*[@data-testid="permalink-generated-url"]
+  ${PERMA_LOCATION_URL}=  Get Text  xpath://*[@data-testid="permalink-generated-url"]
+  Set Suite Variable  ${PERMA_LOCATION_URL}
+
+Go to permalink
+    [Tags]  HappyPath
+    user goes to url  ${PERMA_LOCATION_URL}
+    user waits for page to finish loading
+    user waits until h1 is visible  '${SUBJECT_NAME}' from '${PUBLICATION_NAME}'
+    user checks page does not contain   WARNING - The data used in this permalink may be out-of-date.
+
+Return to Admin to start creating an amendment
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+    user waits until h1 is visible   Dashboard
+    user waits until page contains title caption  Welcome Bau1
+
+Create amendment
+    [Tags]  HappyPath
+    user selects theme and topic from admin dashboard  Test theme  ${TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown   ${AMEND_RELEASE_HEADING}  ${accordion}
+    ${details}=  user gets details content element  ${AMEND_RELEASE_HEADING}  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    ${edit_button}=  get child element  ${details}  xpath:.//button[text()="Amend this release"]
+    user clicks element   ${edit_button}
```

Comment:
Rather than these two lines, you can do `user clicks button | Amend this release |   ${details}`